proto/storage/service_identity.proto

```diff
@@ -33,6 +33,7 @@ enum ServiceType {
   SCANNER_V4_INDEXER_SERVICE = 13;
   SCANNER_V4_MATCHER_SERVICE = 14;
   SCANNER_V4_DB_SERVICE = 15;
+  SCANNER_V4_SERVICE = 16;
```

Comment:
nit: since this is a special case used for testing only, may help to have a comment here in the proto calling that out.

Reply:
I'd rather not comment it here, as I fear it may not be deleted if we do decide to support this in prod. I will change the name to COMBO, though, to make its purpose more obvious, though

Reply:
Actually, I'll keep it as-is and just comment it's for combo-mode. When adding COMBO, then the cert verification library expects the URL to be `scanner-v4-combo.svc` which I think is clunky compared to just `scanner-v4.svc`